micromamba/tests/test_list.py

```diff
@@ -209,3 +210,32 @@ def test_regex(tmp_home, tmp_root_prefix, tmp_xtensor_env, quiet_flag):
     filtered_res = helpers.umamba_list("^xt", "--json", quiet_flag)
     filtered_names = sorted([i["name"] for i in filtered_res])
     assert filtered_names == ["xtensor", "xtl"]
+
+
+@pytest.mark.parametrize("revisions_flag", ["", "--revisions"])
+def test_revisions(revisions_flag):
+    env_name = "myenv"
+
+    helpers.create("-n", env_name, "python=3.8")
+    helpers.install("-n", env_name, "xeus=2.0")
+    helpers.update("-n", env_name, "xeus=4.0")
+    helpers.uninstall("-n", env_name, "xeus")
+    res = helpers.umamba_list("-n", env_name, revisions_flag)
+
+    if revisions_flag == "--revisions":
+        revisions = re.split(r"\d{4}-\d{2}-\d{2}", res)[1:]
```

Comment:
Can you add a comment to explain the parsing and what we should expect (generally speaking)?
(in order to ease future reading of the test).